src/patternfly/components/TextInputGroup/text-input-group.hbs

```diff
@@ -1,6 +1,16 @@
-<div class="pf-c-text-input-group{{#if text-input-group--modifier}} {{text-input-group--modifier}}{{/if}}"
+<div class="pf-c-text-input-group
+  {{#if text-input-group--modifier}}
+    {{text-input-group--modifier}}
+  {{/if}}
+  {{#if text-input-group--IsDisabled}}
+    pf-m-disabled
+  {{/if}}"
   {{#if text-input-group--attribute}}
     {{{text-input-group--attribute}}}
+  {{/if}}
+  {{#if text-input-group--IsDisabled}}
+    aria-label="Disabled text input group example"
+    tabindex="-1"
```

Comment:
I'm actually not sure if this needs `tabindex="-1"`. I tested adding a button to the component and it still allowed me to tab to it, even though the parent component had `tabindex="-1"`. Since this element isn't focusable to begin with, I don't think we need to add this when it's disabled. Sorry for the wrong suggestion :)